src/platforms/native/guides/breakpad/index.mdx

```diff
@@ -8,13 +8,11 @@ description: "Learn about Sentry's integration with Google Breakpad."
 
 [Breakpad](https://chromium.googlesource.com/breakpad/breakpad/) is an open-source multiplatform crash reporting system written in C++ by Google and the predecessor of Crashpad. It supports macOS, Windows and Linux, and features an uploader to submit minidumps to a configured URL right when the process crashes.
 
-As opposed to Crashpad, Breakpad uses in-process crash reporting. This is less robust and has several disadvantages over out-of-process crash reporting. Unless you have integrated Breakpad already, we strongly recommend you to consider using Crashpad instead.
+<Alert level="warning">
```

Comment:
Please don't use Warning here, as the content doesn't rise to this level. Here's out guideline:
- **Note:** use these for useful context, information that should be visually called out from the core content. Use sparingly.
- **Important:** use these for information that's critical to know (this version introduces breaking changes). Use infrequently.
- **Warning:** use these for items that MUST be well understood before proceeding. These should be used very rarely.

Reply:
Fixed in https://github.com/getsentry/sentry-docs/pull/3369